docs/content/creating-tesseracts/create.md

```diff
@@ -129,16 +129,17 @@ current directory is where `tesseract_api.py` is located,
 the full command would be:
 
 ```
-$ tesseract build . -t v1.0.0
+$ tesseract build .
 ```
 
 This is to be interpreted as "build the Tesseract which is located in
-the current directory, and tag it as `v1.0.0`". The name of the Tesseract
-is defined in the `tesseract_config.yaml` file, and it is `helloworld`, so the full
-name of the tesseract we just built is `helloworld:v1.0.0`.
+the current directory". The name of the Tesseract
+is defined in the `tesseract_config.yaml` file, and it is `helloworld`,
+and by default it will be tagged with both the version specified there (`1.0.0`) and `latest`,
+so the full name of the tesseract we just built is `helloworld:1.0.0` or `helloworld:latest`.
```

Comment:
```suggestion
is defined in the `tesseract_config.yaml` file, and it is `helloworld`.
By default it will be tagged with both the version specified there (`1.0.0`) and `latest`,
so the full name of the tesseract we just built is `helloworld:1.0.0` or `helloworld:latest`.
```